wallet/wallet.go

```diff
@@ -327,7 +327,20 @@ func (w *Wallet) activeData(dbtx walletdb.ReadTx) ([]btcutil.Address, []wtxmgr.C
 // finished. The birthday block can be passed in, if set, to ensure we can
 // properly detect if it gets rolled back.
 func (w *Wallet) syncWithChain(birthdayStamp *waddrmgr.BlockStamp) error {
-	// To start, if we've yet to find our birthday stamp, we'll do so now.
+	chainClient, err := w.requireChainClient()
+	if err != nil {
+		return err
+	}
+
+	// We'll wait until the backend is synced to ensure we get the latest
+	// MaxReorgDepth blocks to store. We don't do this for development
+	// environments as we can't guarantee a lively chain.
+	if !w.isDevEnv() {
+		if err := w.waitUntilBackendSynced(chainClient); err != nil {
```

Comment:
Hm, I think I would have a log here. It just makes sense to log before doing a potentially long blocking call.

Reply:
Made it debug to prevent the duplicate log.